presto-jmx/src/main/java/com/facebook/presto/connector/jmx/JmxPeriodicSampler.java

```diff
@@ -69,8 +70,10 @@ public JmxPeriodicSampler(
     @PostConstruct
     public void start()
     {
-        lastDumpTimestamp = roundToPeriod(currentTimeMillis());
-        schedule();
+        if (tableHandles.size() > 0) {
```

Comment:
Ok, indeed. Yesterday I couldn't find if this is documented somewhere, but now I see that behavior in the implementation of `ThreadPoolExecutor`